src/components/Filters/components/Count/Count.tsx

```diff
@@ -9,23 +11,34 @@ const Count: React.FC<NumberProps> = ({
   min,
   max,
   placeholder = 'Count',
+  patternMessage = 'Use only integers',
```

Comment:
I would rename all `patternMesage`s to something more general like `errorMessage` since it is not bound only to pattern. Maybe will be needed update of FE if we will change the API of filters